hpat/__init__.py

```diff
@@ -1,8 +1,9 @@
+from ._version import get_versions
```

Comment:
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

You should put a blank line between each group of imports.